HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -1,5 +1,58 @@
 import FWCore.ParameterSet.Config as cms
 
+# Updating the config (PR #8356)
+def customiseFor8356(process):
+
+    MTRBPSet = cms.PSet(
+        Rescale_eta = cms.double( 3.0 ),
+        Rescale_phi = cms.double( 3.0 ),
+        Rescale_Dz = cms.double( 3.0 ),
+        EtaR_UpperLimit_Par1 = cms.double( 0.25 ),
+        EtaR_UpperLimit_Par2 = cms.double( 0.15 ),
+        PhiR_UpperLimit_Par1 = cms.double( 0.6 ),
+        PhiR_UpperLimit_Par2 = cms.double( 0.2 ),
+        UseVertex = cms.bool( False ),
+        Pt_fixed = cms.bool( False ),
+        Z_fixed = cms.bool( True ),
+        Phi_fixed = cms.bool( False ),
+        Eta_fixed = cms.bool( False ),
+        Pt_min = cms.double( 1.5 ),
+        Phi_min = cms.double( 0.1 ),
+        Eta_min = cms.double( 0.1 ),
+        DeltaZ = cms.double( 15.9 ),
+        DeltaR = cms.double( 0.2 ),
+        DeltaEta = cms.double( 0.2 ),
+        DeltaPhi = cms.double( 0.2 ),
+        maxRegions = cms.int32( 2 ),
+        precise = cms.bool( True ),
+        OnDemand = cms.int32( -1 ),
+        MeasurementTrackerName = cms.InputTag( "hltESPMeasurementTracker" ),
+        beamSpot = cms.InputTag( "hltOnlineBeamSpot" ),
+        vertexCollection = cms.InputTag( "pixelVertices" ),
+        input = cms.InputTag( 'hltL2Muons','UpdatedAtVtx' )
+    )
+
+    def producers_by_type(process, type):
+    	return (module for module in process._Process__producers.values() if module._TypedParameterizable__type == type)
+
+    for l3MPModule in producers_by_type(process, 'L3MuonProducer'):
+	l3MPLabel=l3MPModule._Labelable__label
+	if hasattr(process, l3MPLabel):
+            if hasattr(getattr(process, l3MPLabel), 'GlbRefitterParameters'):
```

Comment:
this part is redundant:
  - you do not need to ask for the module label and re-access the object by label, just use the `l3MPLabel` object
  - you do not need to check that the module actually exist, since you got it from the process itself

something like this is simpler and faster:
```
    for l3MPModule in producers_by_type(process, 'L3MuonProducer'):
        if hasattr(l3MPModule, 'GlbRefitterParameters'):
            l3MPModule.GlbRefitterParameters.RefitFlag = cms.bool(True)
        ...
```